class Error(Exception):
    pass


class FatalError(Error):
    pass
